Failed on ubuntu 18.04

I know it's not easy to read, but that is the message I got - on `build drachtio-server` task. No settings provided, all defaults:

`fatal: [node.zyx.bz]: FAILED! => {"changed": true, "cmd": "./bootstrap.sh && mkdir build && cd $_ && ../configure CPPFLAGS='-DNDEBUG' && make && sudo make install", "delta": "0:04:38.279507", "end": "2019-04-18 18:44:59.259038", "msg": "non-zero return code", "rc": 2, "start": "2019-04-18 18:40:20.979531", "stderr": "configure.ac:29: installing './compile'\nconfigure.ac:16: installing './config.guess'\nconfigure.ac:16: installing './config.sub'\nMakefile.am:2: warning: 'INCLUDES' is the old name for 'AM_CPPFLAGS' (or '*_CPPFLAGS')\n/usr/local/src/drachtio-server/missing: Unknown `--is-lightweight' option\nTry `/usr/local/src/drachtio-server/missing --help' for more information\nconfigure: WARNING: 'missing' script is too old or missing\nconfigure: WARNING: C++17 is not yet standardized, so the checks may change in incompatible ways anytime\nmake: *** [../deps/sofia-sip/libsofia-sip-ua/.libs/libsofia-sip-ua.a] Error 1", "stderr_lines": ["configure.ac:29: installing './compile'", "configure.ac:16: installing './config.guess'", "configure.ac:16: installing './config.sub'", "Makefile.am:2: warning: 'INCLUDES' is the old name for 'AM_CPPFLAGS' (or '*_CPPFLAGS')", "/usr/local/src/drachtio-server/missing: Unknown `--is-lightweight' option", "Try `/usr/local/src/drachtio-server/missing --help' for more information", "configure: WARNING: 'missing' script is too old or missing", "configure: WARNING: C++17 is not yet standardized, so the checks may change in incompatible ways anytime", "make: *** [../deps/sofia-sip/libsofia-sip-ua/.libs/libsofia-sip-ua.a] Error 1"], "stdout": "checking for a BSD-compatible install... /usr/bin/install -c\nchecking whether build environment is sane... yes\nchecking for a thread-safe mkdir -p... /bin/mkdir -p\nchecking for gawk... gawk\nchecking whether make sets $(MAKE)... yes\nchecking whether make supports nested variables... yes\nchecking for style of include used by make... GNU\nchecking for g++... g++\nchecking whether the C++ compiler works... yes\nchecking for C++ compiler default output file name... a.out\nchecking for suffix of executables... \nchecking whether we are cross compiling... no\nchecking for suffix of object files... o\nchecking whether we are using the GNU C++ compiler... yes\nchecking whether g++ accepts -g... yes\nchecking dependency style of g++... gcc3\nchecking whether g++ supports C++11 features by default... yes\nchecking whether g++ supports C++17 features by default... no\nchecking whether g++ supports C++17 features with -std=c++17... yes\nchecking build system type... x86_64-pc-linux-gnu\nchecking host system type... x86_64-pc-linux-gnu\nchecking whether we are using the GNU C++ compiler... (cached) yes\nchecking whether g++ -std=c++17 accepts -g... (cached) yes\nchecking dependency style of g++ -std=c++17... (cached) gcc3\nchecking for gcc... gcc\nchecking whether we are using the GNU C compiler... yes\nchecking whether gcc accepts -g... yes\nchecking for gcc option to accept ISO C89... none needed\nchecking whether gcc understands -c and -o together... yes\nchecking dependency style of gcc... gcc3\nchecking for gawk... (cached) gawk\nchecking for curl-config... /usr/bin/curl-config\nchecking for the version of libcurl... 7.58.0\nchecking whether libcurl is usable... yes\nchecking for curl_free... yes\nchecking that generated files are newer than configure... done\nconfigure: creating ./config.status\nconfig.status: creating Makefile\nconfig.status: creating config.h\nconfig.status: executing depfiles commands\ncd ../deps/boost_1_69_0 && ./bootstrap.sh --with-libraries=thread,system,log,filesystem && ./b2 link=static variant=release stage\nBuilding Boost.Build engine with toolset gcc... tools/build/src/engine/bin.linuxx86_64/b2\nUnicode/ICU support for Boost.Regex?... not found.\nGenerating Boost.Build configuration in project-config.jam...\n\nBootstrapping is done. To build, run:\n\n    ./b2\n    \nTo adjust configuration, edit 'project-config.jam'.\nFurther information:\n\n   - Command line help:\n     ./b2 --help\n     \n   - Getting started guide: \n     http://www.boost.org/more/getting_started/unix-variants.html\n     \n   - Boost.Build documentation:\n     http://www.boost.org/build/doc/html/index.html\n\nPerforming configuration checks\n\n    - default address-model    : 64-bit\n    - default architecture     : x86\n    - native-atomic-int32-supported : yes\n    - native-syslog-supported  : yes\n    - pthread-supports-robust-mutexes : yes\n    - compiler-supports-ssse3  : yes\n    - compiler-supports-avx2   : yes\n    - has_icu builds           : no\n    - lockfree boost::atomic_flag : yes\n\nComponent configuration:\n\n    - atomic                   : not building\n    - chrono                   : not building\n    - container                : not building\n    - context                  : not building\n    - contract                 : not building\n    - coroutine                : not building\n    - date_time                : not building\n    - exception                : not building\n    - fiber                    : not building\n    - filesystem               : building\n    - graph                    : not building\n    - graph_parallel           : not building\n    - iostreams                : not building\n    - locale                   : not building\n    - log                      : building\n    - math                     : not building\n    - mpi                      : not building\n    - program_options          : not building\n    - python                   : not building\n    - random                   : not building\n    - regex                    : not building\n    - serialization            : not building\n    - stacktrace               : not building\n    - system                   : building\n    - test                     : not building\n    - thread                   : building\n    - timer                    : not building\n    - type_erasure             : not building\n    - wave                     : not building\n\n...patience...\n...patience...\n...patience...\n...found 5324 targets...\n...updating 166 targets...\ngcc.compile.c++ bin.v2/libs/system/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/error_code.o\ngcc.archive bin.v2/libs/system/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/libboost_system.a\ncommon.copy stage/lib/libboost_system.a\ngcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/codecvt_error_category.o\ngcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/operations.o\ngcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/path.o\ngcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/path_traits.o\ngcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/portability.o\ngcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/unique_path.o\ngcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/utf8_codecvt_facet.o\ngcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/windows_file_codecvt.o\ngcc.archive bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/libboost_filesystem.a\ncommon.copy stage/lib/libboost_filesystem.a\ngcc.compile.c++ bin.v2/libs/atomic/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/lockpool.o\ngcc.archive bin.v2/libs/atomic/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/libboost_atomic.a\ncommon.copy stage/lib/libboost_atomic.a\ngcc.compile.c++ bin.v2/libs/date_time/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/gregorian/greg_month.o\ngcc.compile.c++ bin.v2/libs/date_time/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/gregorian/greg_weekday.o\ngcc.compile.c++ bin.v2/libs/date_time/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/gregorian/date_generators.o\ngcc.archive bin.v2/libs/date_time/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/libboost_date_time.a\ncommon.copy stage/lib/libboost_date_time.a\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/c_regex_traits.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/cpp_regex_traits.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/cregex.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/fileiter.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/icu.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/instances.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/posix_api.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/regex.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/regex_debug.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/regex_raw_buffer.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/regex_traits_defaults.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/static_mutex.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/w32_regex_traits.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/wc_regex_traits.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/wide_posix_api.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/winstances.o\ngcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/usinstances.o\ngcc.archive bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/libboost_regex.a\ncommon.copy stage/lib/libboost_regex.a\ngcc.compile.c++ bin.v2/libs/thread/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/pthread/thread.o\ngcc.compile.c++ bin.v2/libs/thread/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/pthread/once.o\ngcc.compile.c++ bin.v2/libs/thread/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/future.o\ngcc.archive bin.v2/libs/thread/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/libboost_thread.a\ncommon.copy stage/lib/libboost_thread.a\n...on 100th target...\ngcc.compile.c++ bin.v2/libs/chrono/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/chrono.o\ngcc.compile.c++ bin.v2/libs/chrono/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/thread_clock.o\ngcc.compile.c++ bin.v2/libs/chrono/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/process_cpu_clocks.o\ngcc.archive bin.v2/libs/chrono/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/libboost_chrono.a\ncommon.copy stage/lib/libboost_chrono.a\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/attribute_name.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/attribute_set.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/attribute_value_set.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/code_conversion.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/core.o\nIn file included from ./boost/random/detail/integer_log2.hpp:19:0,\n                 from ./boost/random/detail/large_arithmetic.hpp:19,\n                 from ./boost/random/detail/const_mod.hpp:23,\n                 from ./boost/random/detail/seed_impl.hpp:26,\n                 from ./boost/random/linear_feedback_shift.hpp:27,\n                 from ./boost/random/taus88.hpp:18,\n                 from libs/log/src/core.cpp:30:\n./boost/pending/integer_log2.hpp:7:59: note: #pragma message: This header is deprecated. Use <boost/integer/integer_log2.hpp> instead.\n BOOST_HEADER_DEPRECATED(\"<boost/integer/integer_log2.hpp>\");\n                                                           ^\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/record_ostream.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/severity_level.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/global_logger_storage.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/named_scope.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/process_name.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/process_id.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/thread_id.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/timer.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/exceptions.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/default_attribute_names.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/default_sink.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/text_ostream_backend.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/text_file_backend.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/text_multifile_backend.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/thread_specific.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/once_block.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/timestamp.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/threadsafe_queue.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/event.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/trivial.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/spirit_encoding.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/format_parser.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/date_time_format_parser.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/named_scope_format_parser.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/unhandled_exception_count.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/permissions.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/dump.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/dump_avx2.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/dump_ssse3.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/posix/ipc_reliable_message_queue.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/posix/object_name.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/syslog_backend.o\ngcc.archive bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/libboost_log.a\ncommon.copy stage/lib/libboost_log.a\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/parser_utils.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/init_from_stream.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/init_from_settings.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/settings_parser.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/filter_parser.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/formatter_parser.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/default_filter_factory.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/matches_relation_factory.o\ngcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/default_formatter_factory.o\ngcc.archive bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/libboost_log_setup.a\ncommon.copy stage/lib/libboost_log_setup.a\n...updated 166 targets...\ncd ../deps/sofia-sip && ./bootstrap.sh && ./configure --with-glib=no && make\nbootstrap: checking installation...\nbootstrap: autoconf version 2.69 (ok)\nbootstrap: automake version 1.15.1 (ok)\nbootstrap: aclocal version 1.15.1 (ok)\nbootstrap: libtool not found.\n           You need libtool version 1.5.14 or newer to build sofia-sip from source.\nMakefile:1374: recipe for target '../deps/sofia-sip/libsofia-sip-ua/.libs/libsofia-sip-ua.a' failed", "stdout_lines": ["checking for a BSD-compatible install... /usr/bin/install -c", "checking whether build environment is sane... yes", "checking for a thread-safe mkdir -p... /bin/mkdir -p", "checking for gawk... gawk", "checking whether make sets $(MAKE)... yes", "checking whether make supports nested variables... yes", "checking for style of include used by make... GNU", "checking for g++... g++", "checking whether the C++ compiler works... yes", "checking for C++ compiler default output file name... a.out", "checking for suffix of executables... ", "checking whether we are cross compiling... no", "checking for suffix of object files... o", "checking whether we are using the GNU C++ compiler... yes", "checking whether g++ accepts -g... yes", "checking dependency style of g++... gcc3", "checking whether g++ supports C++11 features by default... yes", "checking whether g++ supports C++17 features by default... no", "checking whether g++ supports C++17 features with -std=c++17... yes", "checking build system type... x86_64-pc-linux-gnu", "checking host system type... x86_64-pc-linux-gnu", "checking whether we are using the GNU C++ compiler... (cached) yes", "checking whether g++ -std=c++17 accepts -g... (cached) yes", "checking dependency style of g++ -std=c++17... (cached) gcc3", "checking for gcc... gcc", "checking whether we are using the GNU C compiler... yes", "checking whether gcc accepts -g... yes", "checking for gcc option to accept ISO C89... none needed", "checking whether gcc understands -c and -o together... yes", "checking dependency style of gcc... gcc3", "checking for gawk... (cached) gawk", "checking for curl-config... /usr/bin/curl-config", "checking for the version of libcurl... 7.58.0", "checking whether libcurl is usable... yes", "checking for curl_free... yes", "checking that generated files are newer than configure... done", "configure: creating ./config.status", "config.status: creating Makefile", "config.status: creating config.h", "config.status: executing depfiles commands", "cd ../deps/boost_1_69_0 && ./bootstrap.sh --with-libraries=thread,system,log,filesystem && ./b2 link=static variant=release stage", "Building Boost.Build engine with toolset gcc... tools/build/src/engine/bin.linuxx86_64/b2", "Unicode/ICU support for Boost.Regex?... not found.", "Generating Boost.Build configuration in project-config.jam...", "", "Bootstrapping is done. To build, run:", "", "    ./b2", "    ", "To adjust configuration, edit 'project-config.jam'.", "Further information:", "", "   - Command line help:", "     ./b2 --help", "     ", "   - Getting started guide: ", "     http://www.boost.org/more/getting_started/unix-variants.html", "     ", "   - Boost.Build documentation:", "     http://www.boost.org/build/doc/html/index.html", "", "Performing configuration checks", "", "    - default address-model    : 64-bit", "    - default architecture     : x86", "    - native-atomic-int32-supported : yes", "    - native-syslog-supported  : yes", "    - pthread-supports-robust-mutexes : yes", "    - compiler-supports-ssse3  : yes", "    - compiler-supports-avx2   : yes", "    - has_icu builds           : no", "    - lockfree boost::atomic_flag : yes", "", "Component configuration:", "", "    - atomic                   : not building", "    - chrono                   : not building", "    - container                : not building", "    - context                  : not building", "    - contract                 : not building", "    - coroutine                : not building", "    - date_time                : not building", "    - exception                : not building", "    - fiber                    : not building", "    - filesystem               : building", "    - graph                    : not building", "    - graph_parallel           : not building", "    - iostreams                : not building", "    - locale                   : not building", "    - log                      : building", "    - math                     : not building", "    - mpi                      : not building", "    - program_options          : not building", "    - python                   : not building", "    - random                   : not building", "    - regex                    : not building", "    - serialization            : not building", "    - stacktrace               : not building", "    - system                   : building", "    - test                     : not building", "    - thread                   : building", "    - timer                    : not building", "    - type_erasure             : not building", "    - wave                     : not building", "", "...patience...", "...patience...", "...patience...", "...found 5324 targets...", "...updating 166 targets...", "gcc.compile.c++ bin.v2/libs/system/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/error_code.o", "gcc.archive bin.v2/libs/system/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/libboost_system.a", "common.copy stage/lib/libboost_system.a", "gcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/codecvt_error_category.o", "gcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/operations.o", "gcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/path.o", "gcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/path_traits.o", "gcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/portability.o", "gcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/unique_path.o", "gcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/utf8_codecvt_facet.o", "gcc.compile.c++ bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/windows_file_codecvt.o", "gcc.archive bin.v2/libs/filesystem/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/libboost_filesystem.a", "common.copy stage/lib/libboost_filesystem.a", "gcc.compile.c++ bin.v2/libs/atomic/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/lockpool.o", "gcc.archive bin.v2/libs/atomic/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/libboost_atomic.a", "common.copy stage/lib/libboost_atomic.a", "gcc.compile.c++ bin.v2/libs/date_time/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/gregorian/greg_month.o", "gcc.compile.c++ bin.v2/libs/date_time/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/gregorian/greg_weekday.o", "gcc.compile.c++ bin.v2/libs/date_time/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/gregorian/date_generators.o", "gcc.archive bin.v2/libs/date_time/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/libboost_date_time.a", "common.copy stage/lib/libboost_date_time.a", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/c_regex_traits.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/cpp_regex_traits.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/cregex.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/fileiter.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/icu.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/instances.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/posix_api.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/regex.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/regex_debug.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/regex_raw_buffer.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/regex_traits_defaults.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/static_mutex.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/w32_regex_traits.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/wc_regex_traits.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/wide_posix_api.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/winstances.o", "gcc.compile.c++ bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/usinstances.o", "gcc.archive bin.v2/libs/regex/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/libboost_regex.a", "common.copy stage/lib/libboost_regex.a", "gcc.compile.c++ bin.v2/libs/thread/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/pthread/thread.o", "gcc.compile.c++ bin.v2/libs/thread/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/pthread/once.o", "gcc.compile.c++ bin.v2/libs/thread/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/future.o", "gcc.archive bin.v2/libs/thread/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/libboost_thread.a", "common.copy stage/lib/libboost_thread.a", "...on 100th target...", "gcc.compile.c++ bin.v2/libs/chrono/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/chrono.o", "gcc.compile.c++ bin.v2/libs/chrono/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/thread_clock.o", "gcc.compile.c++ bin.v2/libs/chrono/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/process_cpu_clocks.o", "gcc.archive bin.v2/libs/chrono/build/gcc-7.3.0/release/link-static/threading-multi/visibility-hidden/libboost_chrono.a", "common.copy stage/lib/libboost_chrono.a", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/attribute_name.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/attribute_set.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/attribute_value_set.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/code_conversion.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/core.o", "In file included from ./boost/random/detail/integer_log2.hpp:19:0,", "                 from ./boost/random/detail/large_arithmetic.hpp:19,", "                 from ./boost/random/detail/const_mod.hpp:23,", "                 from ./boost/random/detail/seed_impl.hpp:26,", "                 from ./boost/random/linear_feedback_shift.hpp:27,", "                 from ./boost/random/taus88.hpp:18,", "                 from libs/log/src/core.cpp:30:", "./boost/pending/integer_log2.hpp:7:59: note: #pragma message: This header is deprecated. Use <boost/integer/integer_log2.hpp> instead.", " BOOST_HEADER_DEPRECATED(\"<boost/integer/integer_log2.hpp>\");", "                                                           ^", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/record_ostream.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/severity_level.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/global_logger_storage.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/named_scope.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/process_name.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/process_id.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/thread_id.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/timer.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/exceptions.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/default_attribute_names.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/default_sink.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/text_ostream_backend.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/text_file_backend.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/text_multifile_backend.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/thread_specific.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/once_block.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/timestamp.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/threadsafe_queue.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/event.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/trivial.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/spirit_encoding.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/format_parser.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/date_time_format_parser.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/named_scope_format_parser.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/unhandled_exception_count.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/permissions.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/dump.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/dump_avx2.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/dump_ssse3.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/posix/ipc_reliable_message_queue.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/posix/object_name.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/syslog_backend.o", "gcc.archive bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/libboost_log.a", "common.copy stage/lib/libboost_log.a", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/parser_utils.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/init_from_stream.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/init_from_settings.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/settings_parser.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/filter_parser.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/formatter_parser.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/default_filter_factory.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/matches_relation_factory.o", "gcc.compile.c++ bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/setup/default_formatter_factory.o", "gcc.archive bin.v2/libs/log/build/gcc-7.3.0/release/link-static/threadapi-pthread/threading-multi/visibility-hidden/libboost_log_setup.a", "common.copy stage/lib/libboost_log_setup.a", "...updated 166 targets...", "cd ../deps/sofia-sip && ./bootstrap.sh && ./configure --with-glib=no && make", "bootstrap: checking installation...", "bootstrap: autoconf version 2.69 (ok)", "bootstrap: automake version 1.15.1 (ok)", "bootstrap: aclocal version 1.15.1 (ok)", "bootstrap: libtool not found.", "           You need libtool version 1.5.14 or newer to build sofia-sip from source.", "Makefile:1374: recipe for target '../deps/sofia-sip/libsofia-sip-ua/.libs/libsofia-sip-ua.a' failed"]}
`

--- a/tasks/main.yml
+++ b/tasks/main.yml
@@ -19,7 +19,7 @@
     apt: 
       name: libtool-bin 
       state: latest
-    when: ansible_distribution == 'Debian'
+    when: ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu'
     ignore_errors: yes          
 
   - name: make source directory writeable

--- a/tasks/main.yml
+++ b/tasks/main.yml
@@ -5,7 +5,7 @@
 
   - name: Install build tools
     apt: 
-        name: ['gcc', 'g++', 'make', 'build-essential', 'git', 'autoconf', 'curl', 'libtool', 'libssl-dev', 'libcurl4-openssl-dev', 'libz-dev']
+        name: ['gcc', 'g++', 'make', 'cmake', 'build-essential', 'git', 'autoconf', 'curl', 'libtool', 'libssl-dev', 'libcurl4-openssl-dev', 'libz-dev']
         state: latest
     when: ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu'
 

